### ISSUE TITLE
App Crashes

### ISSUE THREAD

**Issue by vivekebics** [eneim/toro#267]
I am trying to play video from my local vlc via streaming. URL : http://192.168.1.30:1234

java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.endsWith(java.lang.String)' on a null object reference

**Comment by eneim** [comment]
@vivekebics I don't believe this crash is caused by Toro, can you send here more from crash logcat? ExoPlayer or MediaPlayer should be the one to handle the playback. Also I don't use any "endsWith" method in the library.
